src/borrow.js

```diff
@@ -0,0 +1,53 @@
+const fs = require('fs');
+
+let borrowrawdata = fs.readFileSync('./src/data/borrow.json');  
+let borrow = JSON.parse(borrowrawdata);
+let rawdata = fs.readFileSync('./src/data/books.json');  
+let books = JSON.parse(rawdata);  
+let userrawdata = fs.readFileSync('./src/data/users.json');  
+let users = JSON.parse(userrawdata);
+
+function getborrowList(userID){
+    var book, y;
+    for(var i = 0; i < users.length; i++){
+        if(users[i].id == userID){
+            for(var j = 0; j < borrow.length; j++){
+                //for(var k = 0; k < borrow.length; k++){ //books.json
+                        if(borrow.userID[j].id == userID){
+                            for(var x = 0; x < books.length; x++){
+                                if(borrow.books[y] == books.id[x]){
+                                    book = books[x];
+                                }
+                            }   
+                    }
+                //}
+            }
+        }
+        if(!book){  
+            return {message: 'User doesnt registor in our system. try to inseer the correct user id'};
+        }
+    }
+    return book;
+}
+
```

Comment:
I would rename the method to getBorrowByUserId.
bookID parameter is not necessary since this method should return list of books that the given user rented.